packages/fulfilment-job/src/__tests__/config.spec.js

```diff
@@ -0,0 +1,48 @@
+import AwsMock from 'aws-sdk'
+import config from '../config.js'
+
+describe('config', () => {
+  beforeAll(async () => {
+    AwsMock.SecretsManager.__setResponse('getSecretValue', {
+      SecretString: 'test-ssh-key'
+    })
+    process.env.FULFILMENT_FILE_SIZE = 1234
+    process.env.FULFILMENT_FTP_HOST = 'test-host'
+    process.env.FULFILMENT_FTP_PORT = 2222
+    process.env.FULFILMENT_FTP_PATH = '/remote/share'
+    process.env.FULFILMENT_FTP_USERNAME = 'test-user'
+    process.env.FULFILMENT_FTP_KEY_SECRET_ID = 'test-secret-id'
+    process.env.FULFILMENT_S3_BUCKET = 'test-bucket'
+    await config.initialise()
+  })
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe('file', () => {
+    it('provides properties relating to the fulfilment file', async () => {
+      expect(config.file.size).toEqual(1234)
+    })
+  })
+
+  describe('ftp', () => {
+    it('is an asynchronous function to retrieve ftp configuration values', async () => {
+      expect(config.ftp.host).toEqual('test-host')
+      expect(config.ftp.port).toEqual('2222')
+      expect(config.ftp.path).toEqual('/remote/share')
+      expect(config.ftp.username).toEqual('test-user')
+      expect(config.ftp.privateKey).toEqual('test-ssh-key')
```

Comment:
might be slightly more terse to do:
```
expect (config.ftp).toEqual(
  expect.objectContaining({
    host: 'test-host',
    port: 2222,
    path: '/remote/share'
    username: 'test-user',
    privateKey: 'test-ssh-key'
  })
```
Not much in it though

Reply:
Good point - not sure why I didn't do it like this in the first place!  Updated and I've added some additional expectations as I refactored the FTP configuration object late in the day and didn't add the additional tests for this.